app/views/subscriptions/_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= simple_form_for subscription do |f| %>
   <div class="form-container__body">
-    <p>But there's good news! If you'd like to provide us with your email, we will notify you once we begin service in your area:</p>
-    <%= f.input :email, label: false %>
-    <%= f.input :zipcode, as: :hidden %>
+    <p><%= t(".prompt") %></p>
+    <%= f.input :email %>
+    <%= f.input :zipcode %>
```

Comment:
@seanpdoyle I have no issue with the visibility of the zipcode input. I didn't add that `as: :hidden`. (Not sure where that came from if it's not intended to be there.)

I did remove the label however, and your point about accessibility is valid. I have no issue keeping it around.